Live/Classes/HUD/HUDBar.cpp

```diff
@@ -0,0 +1,42 @@
+#include "HUDBar.h"
+
+USING_NS_CC;
+
+HUDBar::HUDBar(const std::string& _title, double max, int x, int y) {
+    // Set properties
+    HUDBar::m_title = _title;
+    HUDBar::m_maxValue = max;
+    HUDBar::m_currValue = max;
+
+    // Create, customize, and add label
+    m_label = Label::createWithSystemFont(m_title.c_str(), "Ubuntu", 12);
```

Comment:
are there other system fonts we can use? idk if ubuntu would be available on all systems.

Reply:
Looking in to. Loading other fonts had errors.